### ISSUE TITLE
Sugerencia

### ISSUE THREAD

**Issue by Credits125** [jordyamc/UKIKU#12]
Agreguen que recuerde en que minuto quedó el capitulo y se reanude, vi una App de anime que hace eso. No se si sea posible.

**Comment by RedTestament** [comment]
Sí, usas un reproductor externo como el MX o VLC,  y ese se encarga de recordar la ultima posición del video reproducido, no se si eso te sirva de algo por el momento?
